tools/mage/js.go

```diff
@@ -63,20 +63,20 @@ func runYarnV(args ...string) error {
 	return sh.RunV("npx", append([]string{"yarn"}, args...)...)
 }
 
-func (Js) runYarnCommand(cmd string, args ...string) error {
+func runYarnCommand(cmd string, args ...string) error {
```

Comment:
Revert, this is incorrect. These commands must be run in root package, not anywhere else.
If you acknowledge that, you can run `Js.<cmd>`, but there must be a namespace